ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -300,6 +300,15 @@ echo "$(date +%s)" > "${SHARED_DIR}/TEST_TIME_INSTALL_START"
 openshift-install --dir="${dir}" create manifests &
 wait "$!"
 
+# aws-outpost code - begin
+if [[ "${CLUSTER_TYPE}" == 'aws' ]] && [[ -f "${SHARED_DIR}/outpost_stack" ]]; then
+  ls -l "${SHARED_DIR}"
+  cat "${SHARED_DIR}/install-config.yaml"
+  sed -i "s/$(cat $SHARED_DIR/prv_subn)/$(cat $SHARED_DIR/outpost_prv_subn)/" $dir/openshift/99_openshift-cluster-api_worker-machineset-0.yaml
```

Comment:
Does this manifests contain the private subnet in `prv_subn` ? 
And I don't see `worker-machineset-1` and `worker-machineset-2` here, is it intentional?

Reply:
there is no worker-machineset-X for each worker..  what do u mean.    worker-machineset-0 is for all workers.